.atlascli.goreleaser.yml

```diff
@@ -65,7 +70,7 @@ nfpms:
 checksum:
   name_template: checksums.txt
 snapshot:
-  name_template: "{{ .Version }}-next"
+  name_template: "{{ .Env.VERSION_GIT }}-next"
```

Comment:
For unstable releases we cannot use `{{ .Version }}` as it is `0.0.0`, once we make the first release `atlascli/` or` mongocli/` release this will change and we would be able to use `{{ .Version }} ` again.